src/touch/touch.rs

```diff
@@ -18,7 +18,9 @@ extern crate uucore;
 
 use filetime::*;
 use std::fs::{self, File};
-use std::io::{self, Error};
+#[cfg(not(windows))]
+use std::io;
+use std::io::{Error};
```

Comment:
Sorry, I missed this one.  Can you remove the braces around `Error`?

Reply:
👍